tests/tests_pytorch/accelerators/test_mps.py

```diff
@@ -162,3 +162,14 @@ def to(self, *args, **kwargs):
 
     assert batch.text.type() == "torch.mps.LongTensor"
     assert batch.label.type() == "torch.mps.LongTensor"
+
+
+@RunIf(mps=True)
+def test_data_is_not_changed_after_move_to_mps_device():
+    trainer = Trainer(accelerator="mps", devices=1)
```

Comment:
I suggest unit-testing the `move_data_to_device` function only, instead of instantiating an entire trainer